app/assets/javascripts/analytics/static-tracker.js

```diff
@@ -78,6 +79,26 @@
       tracker.setDimension(customVar[1], customVar[3], customVar[2], customVar[4]);
     }
 
+    function setDimensionsFromMetaTags() {
```

Comment:
Is there any reason this method can't be added to the prototype? That would save the awkward `apply(this)` line above and would better follow the approach we suggest in the JavaScript styleguide.